ui/mymints/src/main/java/com/solanamobile/mintyfresh/mymints/composables/MyMints.kt

```diff
@@ -86,6 +87,10 @@ fun NavGraphBuilder.myMintsScreen(
                         identityName = appName,
                         activityResultSender = activityResultSender
                     )
+                    SettingsButton(
```

Comment:
Did you get this in a design from Andrew? Or just wing it.

One thing I was thinking is instead of a whole button for a screen, what if we did an overflow menu that just had the doc options?

Reply:
This is approved by Andrew L